app/Console/Kernel.php

```diff
@@ -23,6 +23,12 @@ protected function schedule(Schedule $schedule)
         $schedule->command('backup:clean')->daily()->at('01:00');
         $schedule->command('backup:run')->daily()->at('02:00');
         $schedule->command('horizon:snapshot')->everyFiveMinutes();
+        $schedule->command('post-article-to-twitter')
+            ->everyMinute()
+            ->when(function () {
+                // ~2 posts every 24 hours.
+                return random_int(1, 1440) <= 2;
```

Comment:
I'm a bit confused here as to how this translates to the comment above. It just seems like an odd chance to hit the 1 or the 2 from this random int? How does this translate to the ~2 evert 24 hours?

Reply:
We'll generate a random int between 1 and 1440 every minute and there are 1440 minutes in 24 hours. 

Assuming there were an even chance of each number being generated, we could assume that in a 24 hour period each number between 1 and 1440 would be returned once.

I know in practice this is unlikely. Some days we will send 0 tweets, and others we may send 5 and maybe we don't want tweets going out in the middle of the night.

Open to suggestions here.

Reply:
Yeah I think we need something more practical here. I think it might be best that we just pick 2-3 fixed moments during the day and then pick the first untweeted article.